utils/librespeed-cli/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2022 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=librespeed-cli
+PKG_VERSION:=1.0.10
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/librespeed/speedtest-cli/tar.gz/v${PKG_VERSION}?
+PKG_HASH:=904ec84f41714d5a7ab778534d332219cd254fdd0f97cc33ebb3540d31fb802c
+
+PKG_MAINTAINER:=Michal Vasilek <michal.vasilek@nic.cz>
+PKG_LICENSE:=LGPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/speedtest-cli-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/librespeed/speedtest-cli
```

Comment:
So the final binary will be called as `speedtest-cli`, which conflicts with the existing package `python3-speedtest-cli`.
You should define `CONFLICTS:=python3-speedtest-cli`, or rename the binary to `librespeed-cli` like [Arch Linux](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=librespeed-cli).

Reply:
Thank you for noticing this. It's unfortunate the librespeed upstream chose to use a conflicting name, I will add the conflicts.

Reply:
No, `speedtest-cli` is the module name. I found upstream also called it as `librespeed-cli`, see [releases](https://github.com/librespeed/speedtest-cli/releases/tag/v1.0.10).

Reply:
I can see that now in their dockerfile and makefile. In that case, I will open a PR to rename it.

Reply:
Thank you!

Reply:
#19947